createdisk.sh

```diff
@@ -213,6 +213,8 @@ if ! which ${QEMU_IMG}; then
     sudo yum -y install /usr/bin/qemu-img
 fi
 
+# This random_string is created by installer and added to each resource type so
+# to maintain consistancy, we are also using random_string here.
```

Comment:
"consistency". But I don't understand the clarification the comment is trying to give :( Are you saying internally, the installer calls it "random_string", and that we are using the same name?

Reply:
yes Installer also using this as `random_string` so we are using same.

Reply:
This is the important part, that this naming is used by the installer, so I would spell it explicitly in the comment.

Reply:
No change in the comment?